test_guru/app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Users::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
+
+  # GET /resource/sign_in
+  # def new
+  #   super
+  # end
+
+  # POST /resource/sign_in
+  def create
+    super
+    if current_user.full_name.present?
```

Comment:
Можно было записать короче
```suggestion
    flash[:notice] = "Привет, #{current_user.full_name.presence || current_user.email}"
```